packages/nouns-contracts/contracts/governance/fork/newdao/governance/NounsDAOLogicV1Fork.sol

```diff
@@ -339,13 +339,18 @@ contract NounsDAOLogicV1Fork is UUPSUpgradeable, ReentrancyGuardUpgradeable, Nou
 
     /**
      * @notice Internal function that reverts if the governance is not active yet. Governance becomes active as soon as
-     * one of these conditions is met:
+     * the forking period ended and one of these conditions is met:
      * 1. All tokens are claimed
      * 2. The delayed governance expiration timestamp is reached
      */
     function checkGovernanceActive() internal view {
-        if (block.timestamp < delayedGovernanceExpirationTimestamp && nouns.remainingTokensToClaim() > 0)
+        if (block.timestamp < nouns.forkingPeriodEndTimestamp()) {
```

Comment:
I think it could be slightly better if this forking period revert had its own custom error, to more clearly set the two reasons apart. wdyt?

Reply:
@eladmallel done